src/buffers.ts

```diff
@@ -68,14 +71,17 @@ export default class BufferList extends BasicList {
     })
   }
 
-  public async loadItems(_context: ListContext): Promise<ListItem[]> {
+  public async loadItems(context: ListContext): Promise<ListItem[]> {
     const { nvim } = this
     const bufnrAlt = Number(await nvim.call('bufnr', '#'))
     const content = await nvim.call('execute', 'ls') as string
+    const isArgs = context.args.indexOf('--args') !== -1
+    const args = isArgs ? await nvim.call('execute', 'ar') as string : ''
```

Comment:
You should use `await nvim.eval("map(argv(), 'bufnr(v:val)')") as number[]` to get list of bufnrs for arguments.